clients/privacy-center/public/fides-js-demo.html

```diff
@@ -177,6 +177,30 @@ <h2>Property id</h2>
       } else {
         window.addEventListener("FidesInitialized", onInitialized);
       }
+
+      window.addEventListener("FidesUIShown", () => {
+        // Log event timing
+        const fidesLoaded = performance
+          .getEntriesByType("resource")
+          .find((entry) => entry.name.includes("fides"));
+        const fidesEvents = performance
+          .getEntriesByType("mark")
+          .filter((entry) => entry.name.includes("Fides"));
+
+        const fidesTiming = {};
+        fidesTiming[
+          `fides.js (${(fidesLoaded.encodedBodySize / 1000).toFixed(2)} kB)`
+        ] = {
+          "Time (ms)": parseFloat(fidesLoaded.responseEnd.toFixed(2)),
```

Comment:
Can `fidesLoaded` be null?

Reply:
only if `fides.js` didn't load on the demo page, and then we have bigger fish to fry 😄